website/project/metadata/prereg-prize-test.json

```diff
@@ -0,0 +1,416 @@
+{
```

Comment:
This file appears to be an actual schema in actual use. So why is the file named "*-test"?